server/adaptors/integrations/__data__/repository/aws_elb/aws_elb-1.0.0.json

```diff
@@ -0,0 +1,57 @@
+{
+    "name": "aws_elb",
+    "version": "1.0.0",
+    "displayName": "AWS ELB",
+    "description": "Nginx HTTP server collector",
+    "license": "Apache-2.0",
+    "type": "logs",
+    "author": "John Doe",
+    "sourceUrl": "https://github.com/",
```

Comment:
We should update this (+ author) to the actual repository that we sourced this from